src/main/kotlin/com/terraformation/backend/accelerator/api/ParticipantProjectSpeciesController.kt

```diff
@@ -87,7 +90,58 @@ class ParticipantProjectSpeciesController(
 
   @ApiResponse200
   @ApiResponse404
-  @PutMapping("/{participantProjectSpeciesId}")
+  @GetMapping("/species/{speciesId}/projects")
+  @Operation(
+      summary =
+          "Gets all participant projects associated to a species with active deliverable information if applicable.")
+  fun getProjectsForSpecies(
+      @PathVariable speciesId: SpeciesId,
+      @RequestParam organizationId: OrganizationId
```

Comment:
Species are per-organization so you should always be able to derive the organization ID from the species ID; no need to make the client pass it in.